openmetadata-ui/src/main/resources/ui/src/styles/x-master.css

```diff
@@ -1117,15 +1117,40 @@ code {
   padding: 4px 8px;
 }
 
+/* Antd Modal CSS  start */
+
 .ant-modal-content {
-  border-radius: 6px;
+  border-radius: 8px;
+  box-shadow: 2px 4px 12px rgba(0, 0, 0, 0.2);
 }
 
 .ant-modal-header {
   border-top-left-radius: 6px;
   border-top-right-radius: 6px;
+  border-bottom: 1px solid #dde3ea;
+}
+
+.ant-modal-footer {
+  padding: 16px 23px;
+  border-top: 1px solid #dde3ea;
 }
 
+.ant-modal-close {
```

Comment:
we are directly modifying antd classes, is it international? if yes please create a separate modal.less file for overwriting antd modal, as in future we are removing x-master.css file

Reply:
ok, will created new file